features/logging/collector.feature

```diff
@@ -42,12 +54,25 @@ Feature: collector related tests
       | component=<%= cb.collection_type %> |
     And evaluation of `@pods.map {|n| n.node_ip}.uniq` is stored in the :node_ips clipboard
     #And evaluation of `cluster_logging('instance').fluentd_ready_pods.map(&:node_ip)` is stored in the :node_ips clipboard
+    #A workaround to https://bugzilla.redhat.com/show_bug.cgi?id=1776594
+    And I wait for the ".operations" index to appear in the ES pod with labels "es-node-master=true"
+    Given I perform the HTTP request on the ES pod with labels "es-node-master=true":
+      | relative_url | .operations.* |
+      | op           | DELETE        |
+    Then the step should succeed
+    Given I perform the HTTP request on the ES pod with labels "es-node-master=true":
+      | relative_url | .project.* |
+      | op           | DELETE     |
+    Then the step should succeed
+    #Workaround end
     And I wait for the ".operations" index to appear in the ES pod with labels "es-node-master=true"
     Given I get the ".operations" logging index information from a pod with labels "es-node-master=true"
     And the expression should be true> cb.index_data['docs.count'] > "0"
-    And I perform the HTTP request on the ES pod with labels "es-node-master=true":
+    Given I wait up to 300 seconds for the steps to pass
```

Comment:
@anpingli an example of step `I wait up to xxx seconds for the steps to pass:`  https://github.com/openshift/verification-tests/blob/master/features/logging/cluster_logging_operator.feature#L29-L36